server/src/Repositories/genericRepository.js

```diff
@@ -0,0 +1,62 @@
+class Repository {
```

Comment:
Why not separate functions instead of classes? We can pass model as a prop to every function.

Reply:
Created separate functions.